dbt-cta/monitoring/models/1_intermediate/composer_logs/int_parse_scheduler_logs.sql

```diff
@@ -79,6 +79,13 @@ with
 
         from unpack_values
     )
+    , filter_unused_dag as (
+        select * from cast_data_types
+        where dag_id not in ('dbt_monitoring') 
```

Comment:
```suggestion
        where dag_id not in ('airflow_monitoring', 'dbt_monitoring') 
```
This is redundant with our log sink exclusion filter, but can we filter out `airflow_monitoring` here as well so it's cute and consistent?

Reply:
for sure, done!